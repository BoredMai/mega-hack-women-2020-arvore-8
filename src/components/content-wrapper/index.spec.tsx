import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import ContentWrapper from '.';

describe('<ContentWrapper>', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(
      <ContentWrapper>
        <div>Content Wrapper</div>
      </ContentWrapper>,
    ) as ShallowWrapper;
  });

  it('renders properly', () => {
    expect(wrapper.getElement()).toMatchSnapshot();
  });
});
