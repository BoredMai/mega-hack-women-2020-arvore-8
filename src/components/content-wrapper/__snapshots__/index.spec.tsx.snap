// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentWrapper> renders properly 1`] = `
<div
  className="content-wrapper"
>
  <div>
    Content Wrapper
  </div>
</div>
`;
